src/data.toml

```diff
@@ -1,17 +1,18 @@
 ["1196075698301968455"] # pyoncord, responses for triggered intents
 undetailed_question = "The information you provided is not detailed enough to answer your question.\nPlease provide more information or rephrase your question."
-get_addons = "Bunny offers compatibility with Vendetta's plugins and themes, since it is what Bunny is based on:\n\n- For plugins, you can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\n- For themes, you may install any Vendetta themes you like. Revenge themes are also supported, excluding custom fonts support.\n\nYou may find a curated list of broken plugins in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
+get_addons = "Bunny offers compatibility with Vendetta (discontinued) & Revenge's(Discontinued) plugins and themes, since Vendetta is what Bunny is based on:\n\n### For plugins:\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\n- Alternatively you can kindly, request for other plugins that you may spot others using in the server that aren't available in the plugin web list.\n### For themes:\n\n- You may install any Vendetta themes you like, preferably more recent ones are recommended as older themes are likely outdated.\n- Revenge themes are also supported, excluding custom fonts support.\n- You can install themes from Enmity as well but theme's colours may be different  depending on your device platforms considering discord uses different strings for both android & iOS.\n\nYou may find a curated list of broken plugins in https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
```

Comment:
this has inconsistent spacing and casing in various positions, and perhaps is a bit much to read

Reply:
got it

Reply:
I will summarise it in a condensed form then

Reply:
done